code/game/mecha/working/ripley.dm

```diff
@@ -195,6 +195,9 @@
 			O.forceMove(drop_location())
 
 /obj/mecha/working/ripley/proc/update_pressure()
+	if(thrusters_active)
+		return // Don't calculate this if they have thrusters on, this is calculated right after domove because of course it is
```

Comment:
not sure how I feel about this proc in general, updating `step_in` in a proc called `update_pressure()` seems like bad practice. This is definitely a bandaid fix

Reply:
Yeah, I'm not a huge fan of anything related to mechcode either, it's just sequenced awfully so there's no other option that won't change behavior.
It's a bit bandaid-y, but the alternative is just reafactoring mechs